examples/mqtt-paho/main.c

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (C) 2019
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       MQTT-paho example
+ *
+ * @author      Javier FILEIV <javier.fileiv@gmail.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <thread.h>
+#include <mqtt.h>
+#include <log.h>
+#include "msg.h"
+#include "lwip.h"
```

Comment:
It would be great if we could get rid of lwip specific includes and only use generic ones. This would allow to use paho-mqtt on top of any IP stack: GNRC or whatever.